src/item.cpp

```diff
@@ -4330,6 +4422,10 @@ const itype * item::ammo_data() const
         return !contents.empty() ? contents.front().ammo_data() : nullptr;
     }
 
+//    if( is_bandolier() ) {
+//        return !contents.empty() ? contents.front().ammo_data() : nullptr;
+//    }
```

Comment:
Please don't check in commented out code. If it is not used, remove it. One can always get it back from the git history.